classes/class_node.rst

```diff
@@ -43,7 +43,7 @@ Finally, when a node is freed with :ref:`Object.free<class_Object_method_free>`
 Tutorials
 ---------
 
-- :doc:`../getting_started/step_by_step/scenes_and_nodes`
+- :doc:`../getting_started/step_by_step/nodes_and_scenes`
```

Comment:
Classref changes should be omitted from this PR and made in the main repo.

Reply:
I have to do both because otherwise the CI will error out: it treats all warnings and issues as stopping errors.